apps/server/src/connect/connectApiExpress.js

```diff
@@ -1,229 +1,231 @@
-    res.send(ret)
-  })
+    const ret = await req.connectApi.addMember(req.body);
+    res.send(ret);
+  });
   app.put(`${ApiEndpoints.MEMBERS}/:member_guid`, async (req, res) => {
     // res.send(require('./stubs/member.js'))
     // return;
-    const ret = await req.connectApi.updateMember(req.body)
-    res.send(ret)
-  })
+    const ret = await req.connectApi.updateMember(req.body);
+    res.send(ret);
+  });
   app.get(`${ApiEndpoints.MEMBERS}/:member_guid`, async (req, res) => {
     // res.send(require('./stubs/member.js'))
     // return;
-    const ret = await req.connectApi.loadMemberByGuid(req.params.member_guid)
-    res.send(ret)
+    const ret = await req.connectApi.loadMemberByGuid(req.params.member_guid);
+    res.send(ret);
     // res.sendFile(__dirname + '/stubs/member.json')
-  })
+  });
   app.get(
     `${ApiEndpoints.MEMBERS}/:member_guid/credentials`,
     async (req, res) => {
       // res.send(require('./stubs/member_credentials.js'))
       // return;
       const ret = await req.connectApi.getMemberCredentials(
-        req.params.member_guid
-      )
-      res.send(ret)
-    }
-  )
+        req.params.member_guid,
+      );
+      res.send(ret);
+    },
+  );
   app.get(
     `${ApiEndpoints.MEMBERS}/:member_guid/oauth_window_uri`,
     async (req, res) => {
-      const ret = await req.connectApi.getOauthWindowUri(req.params.member_guid)
-      res.send({ oauth_window_uri: ret })
+      const ret = await req.connectApi.getOauthWindowUri(
+        req.params.member_guid,
+      );
+      res.send({ oauth_window_uri: ret });
       // res.sendFile(__dirname + '/stubs/member.json')
-    }
-  )
+    },
+  );
   app.delete(`${ApiEndpoints.MEMBERS}/:member_guid`, async (req, res) => {
-    res.sendFile(path.join(__dirname, '/stubs/member.json'))
+    res.sendFile(path.join(__dirname, "/stubs/member.json"));
     // let ret = await req.connectApi.deleteMember(req.params.member_guid)
     // res.send(ret)
-  })
+  });
   app.get(
     `${ApiEndpoints.INSTITUTIONS}/:institution_guid/credentials`,
-    getInstitutionCredentialsHandler
-  )
+    getInstitutionCredentialsHandler,
+  );
 
-  app.get(`${ApiEndpoints.INSTITUTIONS}/favorite`, favoriteInstitutionsHandler)
+  app.get(`${ApiEndpoints.INSTITUTIONS}/favorite`, favoriteInstitutionsHandler);
 
   app.get(
     `${ApiEndpoints.INSTITUTIONS}/:institution_guid`,
-    getInstitutionHandler
-  )
-  app.get(ApiEndpoints.INSTITUTIONS, getInstitutionsHandler)
-  app.get('/jobs/:member_guid', async (req, res) => {
-    if (['mx_int', 'mx'].includes(req.context.aggregator)) {
-      if (req.params.member_guid === 'null') {
-        res.send({ job: { guid: 'none', job_type: AGGREGATION_JOB_TYPE } })
-        return
+    getInstitutionHandler,
+  );
+  app.get(ApiEndpoints.INSTITUTIONS, getInstitutionsHandler);
+  app.get("/jobs/:member_guid", async (req, res) => {
+    if (["mx_int", "mx"].includes(req.context.aggregator)) {
```

Comment:
This logic needs to be part of the mx adapter package. 

We need something like a "jobRequestHandler" function that will be used if the associated aggregator adapter package has one.

Reply:
This is done.

Reply:
Oh, the implementation is in the mx-adapter fork repo. I'll link the PR.

Reply:
New PR with new request handlers: https://github.com/Universal-Connect-Project/ucw-adapter-mx/pull/7/files#diff-43f6d0f158262b4c1d8deaae15664e2950473810423d33a6a7231211d188d3c8R63-R71